docs/pages/admin-guides/access-controls/access-request-plugins/ssh-approval-msteams.mdx

```diff
@@ -11,6 +11,18 @@ compromising productivity.
 
 ![The Microsoft Teams Access Request plugin](../../../../img/enterprise/plugins/msteams.png)
 
+<Details title="This integration is hosted on Teleport Enterprise (Cloud)" open={false}>
+
+(!docs/pages/includes/plugins/enroll.mdx name="the MsTeams integration"!)
```

Comment:
Is `MsTeams` actually the publicly recognized name of the product, or should we say `Microsoft Teams` here?